site/docs/concepts/web-app.md

```diff
@@ -0,0 +1,272 @@
+Select a specification from the **Files** list to view the JSON.
+
+### Editing captures and collections
+
+When you click **Edit specification** for a capture, you're taken to the **Edit Capture** page.
+
+This page is similar to the [**Create Capture**](#creating-a-capture) page as it was filled out just before the capture was published.
+
+To edit a capture:
+
+1. Edit the connection to the source system, if desired. You can either update fields in the **Endpoint Configuration** section or manually update the JSON in the **Specification Editor**.
+
+:::caution
+You may have to re-authenticate with the source system. Be sure to have current credentials on hand before editing the endpoint configuration.
+:::
+
+2. Use the **Collection Selector** to add or remove collections from the capture, if desired.
+
+   To refresh your connection with the source and see an updated list of possible collections, click the **Rediscover All** button,
+   but be aware that it will overwrite all existing collection selections.
+
+3. When you're done making changes, click **Next.**
+
+   Collection specifications become editable.
+
+4. Use the **Specification Editor** to edit collection specifications, if desired. If you make any changes, click **Next** again.
+
+5. Click **Save and Publish**.
+
+Editing a capture only affects how it will work going forward.
+Data that was captured before editing will reflect the original configuration.
+
+It's important to be mindful of how your edits will effect downstream processes. For more information on the implications of editing, see the [reference documentation](../reference/editing.md).
+
+### Creating a capture
+
+When you click **Create Capture**, you're taken to the Create Capture page.
+In the first view, all available capture connectors are displayed.
+
+Select the tile of the system from which you want to capture data to show the full capture form.
+The form details are specific to the connector you chose.
+
+For detailed steps to create a capture, see the [guide](../guides/create-dataflow.md#create-a-capture).
+
+After you successfully publish a capture, you're given the option to materialize the collections you just captured.
+You can proceed to the materialization, or opt to exit to a different page of the web app.
+
+## Collections page
+
+The **Collections** page shows a read-only table of [collections](./collections.md) to which you have access.
+You can view each collection's specification and see a sample of its data.
+This can help you verify that collection data was captured as expected and that you'll be able to materialize it how you want, and troubleshoot it necessary.
+
+To reveal the **Specification** and **Data Preview** sections, expand **Details** next to a collection name.
+
+The **Specification** section shows the collection specification as JSON in a read-only editor.
```

Comment:
In the _Captures page_ section above, I believe you referred to this segment of the UI as the _Specification window_ and here the term is _Specification section_. Calling out for general awareness. I prefer the term _Specification section_ because the term _window_ has certain implications as far as interfaces are concerned.

Reply:
good catch - I think it would definitely be a beneficial exercise at some point to decide on names for different types of UI elements that we use (or maybe y'all already have concrete terminology, in which case please let me know!)

Reply:
For sure! Travis and I have some _working_ names for aspects of the UI. It would be a great exercise to document those terms and refine them with you.